src/messages/manifestjson.js

```diff
@@ -840,3 +840,12 @@ export const ADMIN_INSTALL_ONLY_PROP_RESERVED = {
     only be used in enterprise add-ons.`),
   file: MANIFEST_JSON,
 };
+
+export const ADMIN_INSTALL_ONLY_REQUIRED = {
+  code: 'ADMIN_INSTALL_ONLY_REQUIRED',
+  message: i18n._(`"/browser_specific_settings/gecko/admin_install_only"
+    property must be set to "true"."`),
```

Comment:
There's an extra " at the end

Reply:
https://github.com/mozilla/addons-linter/pull/5347